paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerConfigurationHandler.kt

```diff
@@ -76,6 +76,11 @@ internal class FlowControllerConfigurationHandler @Inject constructor(
             return
         }
 
+        eventReporter.onInit(
+            configuration = configuration,
+            isDecoupling = initializationMode is DeferredIntent,
+        )
```

Comment:
To be aligned with iOS, the `onInit()` should be called right when the object is being configured, as opposed to when its configuration has succeded.